jobs/build/refresh-images/Jenkinsfile

```diff
@@ -79,17 +80,18 @@ node('openshift-build-1') {
             
             sshagent(['openshift-bot']) {
 
-                update_docker_args = "--bump_release"
-                oit_update_docker_args = ""
-                if ( VERSION_OVERRIDE != "" ) {
+                // default to using the atomic-openshift package version
+                // unless the calller provides a version and release
+                if ( VERSION_OVERRIDE == "" ) {
+                    oit_update_docker_args = "--version auto --bumprelease"
```

Comment:
It is worth some conversation to understand, but now that everything is managed by oit, --bump_release is no longer required and auto version should not imply it. I want our new process to be avoiding specifying a release altogether. This makes oit remove the release label from the distgit Dockerfile. When no release field is defined in the Dockerfile, OSBS will choose a unique value for it on each build. That way, CD gets out of the business of choosing/managing a value in the Dockerfile.

Reply:
Also, this appears to be missing --repo-type signed